qiskit/algorithms/evolvers/classical_methods/scipy_evolver.py

```diff
@@ -0,0 +1,94 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""Parent class for SciPyEvolvers"""
+from abc import ABC
+from typing import List, Tuple, Union
+
+import scipy.sparse as sp
+import numpy as np
+
+from qiskit.algorithms.evolvers.evolution_problem import EvolutionProblem
+
+from ...list_or_dict import ListOrDict
+
+
+class SciPyEvolver(ABC):
```

Comment:
This class really is a bunch of convenience functions right - non operate on self, they could all be static methods. Or perhaps even function in a scipy_evolver_utils module. For me since they are not instance methods having this as some base class seems kinda suspect.